### PR TITLE
Added a Config.Diablo.Skip in charfiles which lets you skip killing diablo if set true

### PR DESCRIPTION
**Pull request by Neverwhite**
Many characters cant kill all seal leaders which wont let you summon Diablo. Or maybe you don't want to kill him. Now you have to option to skip Diablo kill.

Also added a new offset to open Seis seal to fix the cannot open seal problem.

### CODE REVIEW

**Comment by noah-** [comment]
Not a fan of the diablo skip, I would suggest going this route:
https://github.com/kolton/d2bot-with-kolbot/blob/master/d2bs/kolbot/libs/bots/Vizier.js

and add ones for the other seal bosses

**Comment by Neverwhite** [comment]
That is also an option however, you will still wait at star for diablo that won't spawn. If it happens that you have many mobs gathered at star that are imuum you will get hit by them all while waiting there. With the skip you teleport to star and leave straight away.

**Comment by noah-** [comment]
You only wait at star if you do a diablo script, and if your character can't kill all the seal bosses you shouldn't be running that script. The vizier script I linked, only kills vizier and leaves.

**Comment by noah-** [comment]
Hi, so I noticed in the baal config there is a "KillBaal" option, if you change your pullrequest to be analogous to that I will merge it.

**Comment by Neverwhite** [comment]
Ok I have changed the Diablo.Skip into Diablo.KillDiablo and is the same as the baal one. Also changed the line in the charfiles.

File: d2bs/kolbot/libs/bots/Diablo.js [hunk 1]
```diff
@@ -111,8 +111,9 @@ function Diablo() {
 			delay(classid === 394 ? 1000 : 500);
 
 			if (!seal.mode) {
-				if (classid === 394 && Attack.validSpot(seal.x + 15, seal.y)) { // de seis optimization
-					Pather.moveTo(seal.x + 15, seal.y);
+				if (classid === 394 && Attack.validSpot(seal.x + 15, seal.y - 1)) { // de seis optimization
```
Comment (noah-) [inline d2bs/kolbot/libs/bots/Diablo.js]: This change is unrelated to this patch, should not be here.

File: d2bs/kolbot/libs/bots/Diablo.js [hunk 2]
```diff
@@ -111,8 +111,9 @@ function Diablo() {
...
+					//Pather.moveTo(seal.x + 15, seal.y);
```
Comment (noah-) [inline d2bs/kolbot/libs/bots/Diablo.js]: Remove this change.

File: d2bs/kolbot/libs/bots/Diablo.js [hunk 3]
```diff
@@ -217,7 +218,7 @@ function Diablo() {
 		}
 
 		if (this.seisLayout === 1) {
-			Pather.moveTo(7771, 5196);
+			Pather.moveTo(7790, 5200);
```
Comment (noah-) [inline d2bs/kolbot/libs/bots/Diablo.js]: Remove this change.

File: d2bs/kolbot/libs/config/Amazon.js [hunk 4]
```diff
@@ -85,6 +85,7 @@ function LoadConfig() {
 		Config.Diablo.SealWarning = "Leave the seals alone!";
 		Config.Diablo.EntranceTP = "Entrance TP up";
 		Config.Diablo.StarTP = "Star TP up";
+		Config.Diablo.KillDiablo = false;
```
Comment (noah-) [inline d2bs/kolbot/libs/config/Amazon.js]: Move this option right after the first option, set default to true.

File: d2bs/kolbot/libs/config/Amazon.js [hunk 5]
```diff
@@ -487,4 +488,4 @@ function LoadConfig() {
 	Config.AutoBuild.DebugMode = true;			//	Debug mode prints a little more information to console and 
 												//	logs activity to /logs/AutoBuild.CharacterName._MM_DD_YYYY.log
 												//	It automatically enables Config.AutoBuild.Verbose
-}
\ No newline at end of file
+}
```
Comment (noah-) [inline d2bs/kolbot/libs/config/Amazon.js]: Make sure file ending is the same.

**Comment by noah-** [comment]
Please rebase and resubmit this PR.
